cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -252,6 +249,20 @@ def upload_fusion_report_to_scout(
 
         LOG.info("Upload %s fusion report finished!", report_type)
 
+    @staticmethod
+    def _check_dna_cases_ok_for_upload(dna_cases, rna_sample):
+        if not dna_cases:
+            raise CgDataError(
+                "Failed to upload on RNA sample %s since there are no DNA samples linked to it via subject_id"
+                % rna_sample.internal_id,
+            )
+
+        if rna_sample.is_tumour and len(dna_cases) > 1:
+            raise CgDataError(
+                "Failed to upload on RNA tumour sample %s since there are more than one tumour DNA sample linked to it via subject_id"
```

Comment:
```suggestion
                "Failed to upload RNA tumour sample %s since there are more than one tumour DNA sample linked to it via subject_id"
```